### PR TITLE
Stop bundling gRPC certificates with Firestore

### PR DESCRIPTION
**Pull request by var-const**
Now that Firestore depends on gRPC-C++ pod version 0.0.6, which includes its own copy of the certificates, we no longer have to bundle them.

### CODE REVIEW

File: Firestore/Example/Firestore.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1273,7 +1273,6 @@
 				54C9EDEE2040E16300A969CD /* Frameworks */,
 				54C9EDEF2040E16300A969CD /* Resources */,
 				EA424838F4A5DD7B337F57AB /* [CP] Embed Pods Frameworks */,
-				DE5C36328822481F6EB6EF16 /* [CP] Copy Pods Resources */,
```
Comment (var-const) [inline Firestore/Example/Firestore.xcodeproj/project.pbxproj]: I haven't manually touched this file, so I presume all the changes come from `pod update`. Looks like gRPC renamed their dependency on OpenSSL from `openssl` to `openssl_grpc`.

**Comment by paulb777** [comment]
Consider merging #2241 first to avoid creating an external merge conflict.

**Comment by wilhuff** [comment]
@var-const Could you approve my changes on top of your original PR?

File: Firestore/Example/App/iOS/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -84,10 +84,15 @@
       "idiom" : "ipad",
       "size" : "83.5x83.5",
       "scale" : "2x"
+    },
+    {
```
Comment (var-const) [inline Firestore/Example/App/iOS/Images.xcassets/AppIcon.appiconset/Contents.json]: Are changes to this file intentional?

**Pull request merged**